extensions/job_management/standard/requirements/job-management/create/REQ_body.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_job-management_create_body]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/create-body
+part:: The body of the POST request SHALL be in JSON format.
```

Comment:
On one hand, it says in a permission that any encoding is allowed, but here it's required to be JSON?!